forch/faucetizer.py

```diff
@@ -225,13 +225,15 @@ def _initialize_host_ports(self, behavioral_faucet_config):
     def _finalize_host_ports_config(self, behavioral_faucet_config, new_testing_device_vlans):
         testing_port_vlans = new_testing_device_vlans.values()
         testing_port_configured = False
+        has_tail_acl = self._config.tail_acl and self._has_acl(self._config.tail_acl)
```

Comment:
THis doesn't seem consistent with the validate function you created.  Why not just use the validate function here?  Should the check for self._config.tail_acl also be in the validate function?

Reply:
Done